CMakeLists.txt

```diff
@@ -76,17 +76,22 @@ endif (FETCH_DEBUG_SANITIZER)
 
 # Enable CCACHE - compiler caching to reduce build times
 if (FETCH_ENABLE_CCACHE)
+  find_program(SCCACGE_FOUND sccache)
   find_program(CCACHE_FOUND ccache)
 
-  if (CCACHE_FOUND)
+  if (SCCACGE_FOUND)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE sccache)
+    # set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK sccache)
```

Comment:
delete ?

Reply:
Also, there appears to be a typo: 'sscacge'

Reply:
done